src/storage/storage.ts

```diff
@@ -913,4 +914,41 @@ export class Storage implements IStorage {
     const accessData = await this._getValidAccessData();
     return (accessData.walletFlags & WALLET_FLAGS.HARDWARE) > 0;
   }
+
+  /**
+   * Return if the nano contract is registered for the given address based on ncKey.
+   * @param ncId Nano Contract ID.
+   * @returns `true` if registered and `false` otherwise.
+   * @async
+   */
+  async isNanoContractRegistered(ncId: string): Promise<boolean> {
+    return this.store.isNanoContractRegistered(ncId);
+  }
+
+  /**
+   * Get nano contract data.
+   * @param ncId Nano Contract ID.
+   * @returns An instance of Nano Contract data.
+   */
+  async getNanoContract(ncId: string): Promise<INcData | null> {
+    return this.store.getNanoContract(ncId);
+  }
+
+  /**
+   * Register nano contract data instance.
+   * @param ncId Nano Contract ID.
+   * @param ncValue Nano Contract basic information.
+   */
+  async registerNanoContract(ncId: string, ncValue: INcData): Promise<void> {
+    return this.store.registerNanoContract(ncId, ncValue);
+  }
+
+  /**
+   * Unregister nano contract.
+   * @param ncId Nano Contract ID.
+   * @param ncValue Nano Contract basic information.
```

Comment:
What's this parameter?

Reply:
I forgot to remove.

Reply:
Fixed at https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/f7f228e35c65a12e4ab9eade924898eb8c63348b